helm/vitess/templates/_orchestrator-conf.tpl

```diff
@@ -46,6 +46,7 @@ data:
     "HostnameResolveMethod": "none",
     "HTTPAuthPassword": "",
     "HTTPAuthUser": "",
+    "HTTPAdvertise": "orchestrator-headless.{{ $namespace }}:80",
```

Comment:
My goal was to set HTTPAdvertise to bypass any Service, and talk directly Pod-to-Pod. I think we can do that by referring here to the per-Pod DNS entries created in the headless service, so it should look something like:

```
POD_NAME.orchestrator-headless.{{ $namespace }}:3000
```

Reply:
I just tried it out and got this error
`FATAL If specified, HTTPAdvertise must include host name`

Reply:
It didn't like not having `http:`. I'm waiting for my cluster to cycle down so I can test it again.
https://play.golang.org/p/fxedGpvILSf

https://github.com/github/orchestrator/blob/677a004d0374e03e78e87bca417e67f673927fa1/go/config/config.go#L566

Reply:
Adding http seems to have done the trick